scripts/ci/changelog-commit.txt

```diff
@@ -0,0 +1 @@
+891e7d47827e99947e46c82f509e479aa13acf24
```

Comment:
For the initial commit of this file, what is the significance of this SHA? I'm thinking the first changelog generated will start with this commit - is that correct?

Reply:
This is a somewhat arbitrary recent commit in order that there be some overlap between the end of the old changelog and the beginning of the new.